docs/overrides/main.html

```diff
@@ -1,9 +1,12 @@
 {% extends "base.html" %}
 {% block announce %}
   <div style="color: white;">
-    <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-262" style="color: #07e798;">
-      <strong>Prefect 2.6.2 is live!</strong>
-    </a> Support for private GitHub repos, new Monte Carlo collection, improved RRule schedules, and more!
+    <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-263" style="color: #07e798;">
+      <strong>Prefect 2.6.3 is live!</strong>
+    </a> Support for using callable objects as tasks, new 
```

Comment:
Oh I don't really want to highlight this as I do not think we should recommend it :D